src/html.sortable.js

```diff
@@ -685,6 +692,9 @@ var sortable = function (sortableElements, options) {
       if (!dragging || !_listsConnected(sortableElement, dragging.parentElement)) {
         return
       }
+      if (maxItems && _filter(_getChildren(sortableElement), _data(sortableElement, 'items')).length >= maxItems) {
```

Comment:
I think that would cause it's own bugs in other cases? If you have an element that's a child of your sortable but not supposed to be draggable. Maybe that's not a realistic use-case though.